Integrations/ThreatConnect/ThreatConnect.py

```diff
@@ -1299,6 +1299,195 @@ def tc_get_groups():
     )
 
 
+def get_group_request(group_type, group_id):
+    tc = get_client()
+    ro = RequestObject()
+    ro.set_http_method('GET')
+    ro.set_request_uri('/v2/groups/{}/{}'.format(group_type, group_id))
+
+    return tc.api_request(ro).json()
+
+
+def get_group():
+    """
+    retrieve a single Group
+    """
+    group_type = demisto.args().get('group_type')
+    group_id = int(demisto.args().get('group_id'))
+
+    response = get_group_request(group_type, group_id)
+    if group_type == 'adversaries':
+        data = response.get('data', {}).get('adversarie', {})
+    if group_type == 'campaigns':
+        data = response.get('data', {}).get('campaign', {})
+    if group_type == 'documents':
+        data = response.get('data', {}).get('document', {})
+    if group_type == 'emails':
+        data = response.get('data', {}).get('email', {})
+    if group_type == 'events':
+        data = response.get('data', {}).get('event', {})
+    if group_type == 'incidents':
+        data = response.get('data', {}).get('incident', {})
+    if group_type == 'intrusionSets':
+        data = response.get('data', {}).get('intrusionSet', {})
+    if group_type == 'reports':
+        data = response.get('data', {}).get('report', {})
+    if group_type == 'signatures':
+        data = response.get('data', {}).get('signature', {})
+    if group_type == 'threats':
+        data = response.get('data', {}).get('threat', {})
+
+    if response.get('status') == 'Success':
+        contents = {
+            'ID': data.get('id'),
+            'Name': data.get('name'),
+            'Owner': data.get('owner'),
+            'DateAdded': data.get('dateAdded'),
+            'EventDate': data.get('eventDate'),
+            'Status': data.get('status')
+        }
+    else:
+        return_error(response.get('message'))
+
+    context = {
+        'TC.Group(val.ID && val.ID === obj.ID)': contents
+    }
+
+    return_outputs(
+        tableToMarkdown('Group information', contents, removeNull=True),
+        context
+    )
+
+
+def get_group_attributes_request(group_type, group_id):
+    tc = get_client()
+    ro = RequestObject()
+    ro.set_http_method('GET')
+    ro.set_request_uri('/v2/groups/{}/{}/attributes'.format(group_type, group_id))
+
+    return tc.api_request(ro).json()
+
+
+def get_group_attributes():
+    """
+    Retrieve a Group’s Attributes
+    """
+    group_type = demisto.args().get('group_type')
+    group_id = int(demisto.args().get('group_id'))
+    contents = []
+    headers = ['ID', 'Type', 'Value', 'DateAdded', 'LastModified', 'Displayed']
+    response = get_group_attributes_request(group_type, group_id)
+    data = response.get('data', {}).get('attribute', [])
+
+    if response.get('status') == 'Success':
+        for attribute in data:
+            contents.append({
+                'ID': attribute.get('id'),
```

Comment:
is this the id of the group or the attribute

Reply:
ID of the attribute

Reply:
so this doesnt fit here